open-ce/docker_build.py

```diff
@@ -139,21 +138,25 @@ def build_in_container(image_name, output_folder, arg_strings):
     result = _execute_in_container(container_name, cmd)
 
     # Cleanup
-    _stop_container(container_name)
+    #_stop_container(container_name)
 
     if result:
         print("Error executing build in container")
 
     return result
 
-def build_with_docker(output_folder, arg_strings):
+def build_with_docker(output_folder, cuda_versions, arg_strings):
     """
     Create a build image and run a build inside of container based on that image.
     """
     parser = make_parser()
     _, unused_args = parser.parse_known_args(arg_strings)
-
-    result, image_name = build_image()
+    if utils.cuda_level_supported(cuda_versions):
```

Comment:
Since cuda_versions can be a list, we need to verify that every version in the list is supported.

Reply:
build_image will also need to be modified to take a cuda version as an argument. It will need to be able to pick out which build image to use.

Reply:
The arg_strings will need to be modified to only pass in the single CUDA version that will be used.